ui/src/app/utils/runtime-utils.tsx

```diff
@@ -277,17 +279,51 @@ export const useRuntimeStatus = (currentWorkspaceNamespace): [
               throw e;
             }
           }
-        }]);
+        }],
+        [RuntimeStatusRequest.Start, async() => {
+          try {
+            await LeoRuntimeInitializer.initialize({workspaceNamespace: currentWorkspaceNamespace, maxCreateCount: 0, maxPauseCount: 0});
+          } catch (e) {
+            // ExceededActionCountError is expected, as we exceed our create limit of 0.
+            if (!(e instanceof ExceededActionCountError ||
+                e instanceof LeoRuntimeInitializationAbortedError)) {
+              throw e;
+            }
+          }
+        }],
+        [RuntimeStatusRequest.Stop, async() => {
+          try {
+            await LeoRuntimeInitializer.initialize({
+              workspaceNamespace: currentWorkspaceNamespace,
+              maxCreateCount: 0,
+              maxPauseCount: 1,
```

Comment:
If you remove `maxPauseCount` per my other comment, I would consolidate all of these. Only the `resolutionCondition` should need to vary between these different options.